mise.toml

```diff
@@ -10,9 +10,9 @@ yq = "4.44.5"
 # Foundry is a special case because it supplies multiple binaries at the same
 # GitHub release, so we need to use the aliasing trick to get mise to not error
 # The git ref here should be on the `stable` branch.
-forge = "v1.0.0"
-cast = "v1.0.0"
-anvil = "v1.0.0"
+forge = "nightly"
```

Comment:
This should not be nightly but it needs to include the fix for the panic.

Reply:
https://github.com/foundry-rs/foundry/issues/10012

Reply:
@AmadiMichael is working on spearate PR so that we don't need to update the foundry version in this pr.